doc/classes/Vector2.xml

```diff
@@ -326,6 +326,15 @@
 				Returns the result of reflecting the vector from a line defined by the given direction vector [param n].
 			</description>
 		</method>
+		<method name="rotate_toward" qualifiers="const">
+			<return type="Vector2" />
+			<param index="0" name="to" type="Vector2" />
+			<param index="1" name="delta" type="float" />
+			<description>
+				Returns the result of rotating this vector toward [param to], by increment [param delta] (in radians). Passing a negative [param delta] will rotate toward the opposite of [param to].
+				This method supports vectors of different length, with the same behavior as [method slerp]. If the vectors are colinear, this method behaves like [method move_toward]. If [param to] has a length of zero, this method behaves like [method lerp].
```

Comment:
You should write that delta supports only radians.

Reply:
It's already specified that it's in radians, albeit inconsistent with other methods (seems to mostly use ", in radians" rather than "(in radians)", so I updated that.